### ISSUE TITLE
Dragging sprites behaves differently from intended

### ISSUE THREAD

**Issue by espertus** [mit-cml/appinventor-sources#1822]
When testing a development branch, I discovered that Ball/ImageSprite.Dragged behaves differently from how I expected (as the original author) and, more importantly, how the user might expect.

Consider these blocks:
![blocks (2)](https://user-images.githubusercontent.com/661056/64451653-7ef3ee80-d099-11e9-9fc6-1526b38570db.png)

I thought this would cause the sprite (Ball or ImageSprite) to move smoothly (i.e., if the user put down a finger and moved it 10 pixels right, the sprite would move 10 pixels right).

Consider the case of an ImageSprite with a width of 100 pixels and its upper left corner at (100, 100). If the user puts their finger down at (100, 100) and moves it 10 pixels right, curX would be 110, and the ImageSprite would move to (110, 100), which would be the expected behavior.

If the user instead puts their finger at the rightmost edge of the ImageSprite (199, 100) and moves it 10 pixels right, curX would be 209, and the ImageSprite would move to (209, 100), a big jump for a small finger move.

The desired behavior could be obtained with these blocks:
![blocks (3)](https://user-images.githubusercontent.com/661056/64453786-37239600-d09e-11e9-8699-66fd3a1cb4eb.png)

[Our documentation](http://ai2.appinventor.mit.edu/reference/components/animation.html) is accurate about the meaning of the Dragged parameters; however, all of the tutorials I've found get it wrong, including [ours](https://appinventor.mit.edu/explore/sites/all/files/ConceptCards/ai2/DragASprite.pdf). If the sprite is not very wide/high, the first set of blocks works well enough.

At this point, we should probably not change the behavior, but we should acknowledge it in our documentation. I would be happy to add something to the reference documentation.

**Comment by ewpatton** [comment]
@espertus Good catch. If you have time to produce the updated reference documentation that would be great. @kdclang Do you have source files for the concept cards?

**Comment by espertus** [comment]
@ewpatton I was thinking of other changes to the sprite documentation, which I could do at the same time if you think they're a good idea. Right now, there is shared documentation for most of ImageSprite's and Ball's members, which refer to "sprites". I'm wondering if I should split out the documentation, since it might not make sense to a user to refer to a Ball as a sprite.

**Comment by espertus** [comment]
I saw other mistakes in sprite documentation: such as what `Enabled` affects. I'll fix that too.
